tools/config/Tomb1Main_ConfigTool/Resources/Lang/en.json

```diff
@@ -130,6 +130,10 @@
       "Title": "Lara's starting health",
       "Description": "Sets Lara's health value for the beginning of each level."
     },
+    "enable_difficulty": {
+      "Title": "Difficulty Selector",
```

Comment:
Could we change to sentence case so to match the rest of the settings? And the same applies for the French translation.